### ISSUE TITLE
Quest: Bashal'Aran, grell earing not dropping

### ISSUE THREAD

**Issue by ronaldofenomeno** [darrowshire-dev/bugtracker#116]
https://classic.wowhead.com/quest=955/bashalaran
https://classic.wowhead.com/item=5336/grell-earring

according to wowhead, the drop rate is ~ 40%. But its not dropping at all.

**Comment by MoneyControlStockScraper** [comment]
![2021-06-25 15_44_57-Greenshot](https://user-images.githubusercontent.com/84612798/123409887-528dc700-d5cc-11eb-993a-85d6ebd1510f.png)
Killed 3 grell and got 1 earing

**Comment by BugTestR** [comment]
I just did this quest and there are tons of Grell but hardly any Satyr in the area for the follow up quest. Satyr are not dropping the quest item

**Comment by BugTestR** [comment]
Also note that the mobs that drop the quest item: https://classic.wowhead.com/item=5336/grell-earring ,  are not performing their abilities. They are just melee without any spells going off.

Vile Sprite abilities:  https://classic.wowhead.com/npc=2189/vile-sprite#abilities     (couldn't find the correct nature damage amount nor the proc rate in which this mob peforms it's poison attack)
Vile Sprite drops :  https://classic.wowhead.com/npc=2189/vile-sprite#drops        (41% drop rate on the quest item grell earring)

Wild Grell abilities: https://classic.wowhead.com/npc=2190/wild-grell#abilities     (20% attack speed for 1 min. Craze spell)
Wild Grell drops:  https://classic.wowhead.com/npc=2190/wild-grell#drops         (42% drop rate on the quest item grell earring)

**Comment by MoneyControlStockScraper** [comment]
You are right, https://classic.wowhead.com/npc=2189/vile-sprite#abilities does not use its abilities. 

However, https://classic.wowhead.com/npc=2190/wild-grell#abilities does use its abilities

**Comment by BugTestR** [comment]
You're correct wild grell npc is using it's craze ability at ~ 30% HP, re-tested it today.
